composer.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I like to have the `sort-packages` config in Composer, just to keep the dependency list tidy from the get-go. Not a big deal either way https://getcomposer.org/doc/06-config.md#sort-packages

Reply:
Good call. I'll make that change along with adding a support section.